scripts/tracking/train.py

```diff
@@ -0,0 +1,271 @@
+""" SiamRPN train"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import argparse
+import logging
+import os
+import time
+
+import numpy as np
+
+import mxnet as mx
+
+from mxnet import gluon, nd, context, autograd
+from mxnet.contrib import amp
+
+from gluoncv.utils import LRScheduler, LRSequential, split_and_load
+from gluoncv.data.tracking_data.track import TrkDataset
+from gluoncv.model_zoo import get_model
+from gluoncv.loss import SiamRPNLoss
+
+os.system("export MXNET_CUDNN_AUTOTUNE_DEFAULT=0")
```

Comment:
I think this line should be moved to line 13 before mxnet import

Reply:
ok